{
  "id": "CVE-2024-39329",
  "sourceIdentifier": "cve@mitre.org",
  "published": "2024-07-10T05:15:12.097",
  "lastModified": "2024-07-10T05:15:12.097",
  "vulnStatus": "Received",
  "descriptions": [
    {
      "lang": "en",
      "value": "An issue was discovered in Django 5.0 before 5.0.7 and 4.2 before 4.2.14. The django.contrib.auth.backends.ModelBackend.authenticate() method allows remote attackers to enumerate users via a timing attack involving login requests for users with an unusable password."
    }
  ],
  "metrics": {},
  "references": [
    {
      "url": "https://docs.djangoproject.com/en/dev/releases/security/",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://groups.google.com/forum/#%21forum/django-announce",
      "source": "cve@mitre.org"
    },
    {
      "url": "https://www.djangoproject.com/weblog/2024/jul/09/security-releases/",
      "source": "cve@mitre.org"
    }
  ]
}